src/components/CustomMUIComponents/CommonForms.js

```diff
@@ -85,4 +105,87 @@ export const ProgramSelect = ({userData, handleProgramChange}) => (
             ))}
         </Select>
     </FormControl>
-)
\ No newline at end of file
+)
+
+/**
+ * @Author: Jasmin Guay.
+ * Reusable ProfileToggle Toggle input.
+ * @param userData.privateProfile : boolean representing if the student has a privateProfile (true) or not (false).
+ * @param handlePrivacyChange : function handling the changes of userData.privateProfile
+ * @returns {JSX.Element} : The reusable component.
+ */
+export const ProfileToggle = ({userData, handlePrivacyChange}) => (
+    <React.Fragment>
+        <Typography>
+            Profile Privacy
+        </Typography>
+        <FormControlLabel
+            sx={{m: 1, marginY: '5px'}}
+            control={<CustomToggle checked={userData.privateProfile} onChange={handlePrivacyChange} />}
+            label={ <ProfileToggleLabel text={userData.privateProfile ? "Public" : "Private"} />} />
+        <Typography sx={{color: '#DA3A16'}}>
+            Hiding my profile will disable users from adding me to their study groups
+        </Typography>
+    </React.Fragment>
+)
+
+const ProfileToggleLabel = (props) => (
+        <div style={{marginLeft: '8px'}}>
+            <Typography> {props.text}</Typography>
+        </div>
+);
+
+export const CustomToggle = styled((props) => (
+    <Switch
+        focusVisibleClassName=".Mui-focusVisible" disableRipple {...props} />
+))
+
+(({ theme }) => ({
+    width: 40,
+    height: 26,
+    padding: 3,
+    '& .MuiSwitch-switchBase': {
+        padding: 0,
+        margin: 2,
+        transitionDuration: '300ms',
+        color: '#C8C8C8',
+        '&.Mui-checked': {
+            transform: 'translateX(16px)',
+            color: '#C8C8C8',
+            '& + .MuiSwitch-track': {
```

Comment:
I would just put colors in our theme for the toggle 
#912338 for the red and use #C8C8C8 everywhere where there is green just to be consistent